File: SolarPanelAppWs/Model/part.py
class part:
    def __init__(self, _partid, _name, _unitprice, _sum, _maxperstorage):
        self.partId = _partid
        self.name = _name
        self.unitPrice = _unitprice
        self.sum = _sum
        self.maxPerStorage = _maxperstorage
        self.errorCode = 0

    def serialize(self):
        return {"partId" : self.partId, "name" : self.name, "unitPrice" : self.unitPrice, "sum" : self.sum, "maxPerStorage" : self.maxPerStorage, "errorCode" : self.errorCode}

File: SolarPanelAppWF/Part.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SolarPanelAppWF
{
    class Part
    {
        public int partId { get; set; }
        public string name { get; set; }
        public double unitPrice { get; set; }
        public int sum { get; set; }
        public int maxPerStorage { get; set; }
        public int errorCode { get; set; }
    }
}
